packages/server/billing/helpers/removeTeamsLimitObjects.ts

```diff
@@ -1,20 +1,22 @@
+import getKysely from '../../postgres/getKysely'
+import {DB} from '../../postgres/pg'
 import {r} from 'rethinkdb-ts'
 import {RValue} from '../../database/stricterR'
 import {DataLoaderWorker} from '../../graphql/graphql'
 import updateNotification from '../../graphql/public/mutations/helpers/updateNotification'
 
 const removeTeamsLimitObjects = async (orgId: string, dataLoader: DataLoaderWorker) => {
-  const removeJobTypes = ['LOCK_ORGANIZATION', 'WARN_ORGANIZATION']
+  const removeJobTypes = ['LOCK_ORGANIZATION', 'WARN_ORGANIZATION'] as DB['ScheduledJob']['type'][]
```

Comment:
+1 `as const` is a little narrower here, same can be applied to the line below it or any literal tuple

Reply:
I keep forgetting this!